### ISSUE TITLE
Release 10.8.4 missing changelog (?)

### ISSUE THREAD

**Issue by belgattitude** [primefaces/primereact#7335]
### Describe the bug

Hello, the latest version 10.8.4 is available on npm. But I couldn't find the changelog or the releases notes.

Not a dealbreaker, just wanted to mention.

Thanks for your great work !

### System Information

```Shell
Not applicable
```

**Comment by melloware** [comment]
@mertsincan can you click the Release button please?

**Comment by frle10** [comment]
Bump on this :)

**Comment by sja-cslab** [comment]
@melloware you can close this - changelog is there

**Comment by inad9300** [comment]
It's been added with the wrong date, though, I believe, as 10.8.4 is reported as having been released before 10.8.3.

**Comment by melloware** [comment]
yep I can't update the releases only @mertsincan has the permissions.

**Comment by inad9300** [comment]
Still stuck with the incorrect/incomplete changelog :cry:  Wouldn't it be better to leave the ticket open? I fear it being forgotten.

**Comment by inad9300** [comment]
@mertsincan @melloware This issue remains unresolved.

Funnily, what prompted this reminder now is that 10.9.0 and 10.9.1 are reported as having been released in Januay, 2024! 😄  It may be worth looking into automating some changelog checks / generation. Alternatively, please share your time machine with the rest of us!

**Comment by melloware** [comment]
only Mert can fix this...

**Comment by inad9300** [comment]
But could you, please, at least reopen the ticket? 😇  I don't think it's urgent or anything, but it should eventually get corrected.

**Comment by melloware** [comment]
Closing this as I think Mert is too busy to fix this
